runtime/domain.c

```diff
@@ -261,28 +261,28 @@ CAMLexport uintnat caml_minor_heaps_end;
 static CAMLthread_local dom_internal* domain_self;
 
 /*
- * This structure is protected by all_domains_lock
- * [0, participating_domains) are all the domains taking part in STW sections
- * [participating_domains, Max_domains) are all those domains free to be used
+ * This structure is protected by all_domains_lock.  [0, participating_domains)
+ * are all the domains taking part in STW sections.  [participating_domains,
+ * caml_params->max_domains) are all those domains free to be used.
```

Comment:
Nit: I would rather keep the `[foo, bar)` intervals on a single line each for readability, which requires doing some manual line-breaking here.

Reply:
Done in https://github.com/ocaml/ocaml/pull/13272/commits/263be361baad53f6ef36f8774ac8b5d9b1572525.